README.md

```diff
@@ -1,18 +1,33 @@
 # RADAR-Gateway
 
-[![Build Status](https://travis-ci.org/RADAR-base/RADAR-Gateway.svg?branch=master)](https://travis-ci.org/RADAR-base/RADAR-Gateway)
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/79b2380112c5451181367ae16e112025)](https://www.codacy.com/app/RADAR-base/RADAR-Gateway?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=RADAR-base/RADAR-Gateway&amp;utm_campaign=Badge_Grade)
-[![Docker Build](https://img.shields.io/docker/build/radarbase/radar-gateway.svg)](https://cloud.docker.com/swarm/radarbase/repository/docker/radarbase/radar-gateway/general)
+[![Build Status](https://travis-ci.org/RADAR-base/RADAR-PushEndpoint.svg?branch=master)](https
+://travis-ci.org/RADAR-base/RADAR-PushEndpoint)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/79b2380112c5451181367ae16e112025)](https://www.codacy.com/app/RADAR-base/RADAR-PushEndpoint?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=RADAR-base/RADAR-Gateway&amp;utm_campaign=Badge_Grade)
+[![Docker Build](https://img.shields.io/docker/build/radarbase/radar-push-endpoint.svg)](https
+://cloud.docker.com/swarm/radarbase/repository/docker/radarbase/radar-push-endpoint/general)
 
-REST Gateway to the Apache Kafka, similar to the REST Proxy provided by Confluent. In addition, it does authentication and authorization, content validation and decompression if needed. It is available as a [docker image](https://hub.docker.com/r/radarbase/radar-gateway).
+RADAR Push Endpoint that exposes REST interface for push subscription based APIs to the Apache
+ Kafka.
 
 ## Configuration
 
-The [RADAR-Auth] library is used for authentication and authorization of users. Refer to the documentation there for a full description of the configuration options.
+Currently, Garmin is integrated. For adding more services, see the [Extending section](#extending).
+
+```yaml
+   pushIntegration:
+      enabledServices: ["garmin", "service-2", ..., "service-n"]
+      # Push service specific config
+      garmin: 
+        userRepositoryClass: "org.radarbase.push.integration.garmin.user.ServiceUserRepository"
```

Comment:
I'm wondering whether this service user repository should be garmin-specific, as it is a general rest-source-authorizer integration, right?

Reply:
Yes, I could create an abstract class for `GarminUserRepository` and make the service repo extend that and also check the same in the validation.